Development/nmos/video_h265.h

```diff
@@ -0,0 +1,67 @@
+#ifndef NMOS_VIDEO_H265_H
+#define NMOS_VIDEO_H265_H
+
+#include "nmos/media_type.h"
+#include "nmos/sdp_utils.h"
+
+namespace sdp
+{
+    namespace fields
+    {
+        // See https://datatracker.ietf.org/doc/html/rfc7798#page-64
+        //H265 payload mapping
+        const web::json::field<uint32_t> profile_id{U("profile-id")};
+        const web::json::field<uint32_t> level_id{U("level-id")};
+        const web::json::field<uint32_t> interop_constraints{U("interop-constraints")};
+        const web::json::field_as_string sprop_vps{U("sprop-vps")};
+        const web::json::field_as_string sprop_sps{U("sprop-sps")};
+        const web::json::field_as_string sprop_pps{U("sprop-pps")};
+    }
+}
+
+namespace nmos
+{
+    namespace media_types
+    {
+        // H.265 Video
+        // See https://datatracker.ietf.org/doc/html/rfc7798
+        const media_type video_H265{ U("video/H265") };
+    }
+
+
+    // Additional "video/H265" parameters
+    // See https://datatracker.ietf.org/doc/html/rfc7798#section-4.4
+    struct video_h265_parameters
```

Comment:
At the moment, I have been thinking we should capitalize in structs and function names to match the actual media type, like `video_H265_parameters`, as `audio_L_parameters` in existing nmos-cpp, but we could debate this...

Reply:
I agree, changing it!